src/test/java/duke/test/DukeListTest.java

```diff
@@ -0,0 +1,107 @@
+package duke.test;
+
+
+import duke.DukeList;
+import duke.exceptions.DukeInvalidCommandException;
+import duke.exceptions.DukeInvalidDescriptionException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+
+public class DukeListTest {
+
+    private static final char DONE = '\u2713';
+    private static final char NOT_DONE = '\u2717';
+
+
+    @Test
+    public void add_correctInput_addedCorrectly() {
```

Comment:
Very very good test code function namings here using featureUnderTest_testScenario_expectedBehavior(). 👍🏻 In fact i'll improve my code namings as well after looking at how you named them for Week 5's ip